src/runtime_src/core/pcie/tools/xbmgmt/flasher.cpp

```diff
@@ -331,19 +331,22 @@ std::vector<DSAInfo> Flasher::getInstalledDSA()
     {
         // std::cout << "DSA " << dsa.name << ": " << dsa.vendor << " " << dsa.board << " " << dsa.vendor_id << " " << dsa.device_id << "TS: " << dsa.timestamp << std::endl;
         if (!dsa.hasFlashImage || dsa.timestamp == NULL_TIMESTAMP)
-	       continue;
+            continue;
```

Comment:
Looks like nothing changed in this file. Can you please remove the extra spacing and formatting.

Reply:
Fixed extra space and formatting, changes done as per master branch file.

Reply:
Thanks!